tools/codegen/api/dispatcher.py

```diff
@@ -67,7 +67,7 @@ def name(func: FunctionSchema) -> str:
     return cpp.name(func)
 
 def arguments(func: FunctionSchema) -> Sequence[DispatcherArgument]:
-    if local.use_c10_dispatcher() is UseC10Dispatcher.full:
+    if local.use_c10_dispatcher().dispatcher_uses_new_style():
```

Comment:
try to keep an eye out for people merging diffs that use the old style. One way to make sure people notice is by renaming UseC10Dispatcher 🥉

Reply:
(need to keep an eye out because old style uses will not merge conflict, and they will "work" except when they don't)